app/classes/submissions_importer.rb

```diff
@@ -37,120 +37,135 @@ def self.attribute_val(row, attributes, name)
     row['f'][attributes.index(name)]['v']
   end
 
-  def self.create_submissions(data, test_type)
```

Comment:
Are we reworking the format of our submissions? Do I understand this correctly?

Reply:
I moved the function lower in the file, yes.